k9mail-library/src/test/java/com/fsck/k9/mail/K9LibRobolectricTestRunner.java

```diff
@@ -0,0 +1,20 @@
+package com.fsck.k9.mail;
+
+import org.junit.runners.model.InitializationError;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+public class K9LibRobolectricTestRunner extends RobolectricTestRunner {
+
+    public K9LibRobolectricTestRunner(Class<?> testClass) throws InitializationError {
+        super(testClass);
+    }
+
+    @Override
+    protected Config buildGlobalConfig() {
+        return new Config.Builder()
+                .setSdk(21)
+                .setManifest(Config.NONE)
```

Comment:
Yup, this is the runner for k9mail-library, there is a different runner for the app which defines a Manifest path here by default. Besides, you can always override this default config via annotations again.